languages/rust/concepts/numbers/about.md

```diff
@@ -1 +1,13 @@
-TODO: add information on numbers concept
+# Numbers
+
+The very word "computer" derives from "computation", meaning an operation performed on numbers. Computers, and Rust, are highly optimized for performing calculations on numbers.
+
+Rust has several different primitive numeric types, which are appropriate for different use cases. The most basic choice is between [integers](../integers/about.md) and [floating-point numbers](../floating-point-numbers/about.md). There are also a variety of external crates providing non-primitive numeric types for specialized uses.
+
+Rust defines several comparison and mathematical operators which are applied to numbers. If desired, you can apply these operators to your own type by implementing the appropriate trait. The appropriate traits are defined in [`std::ops`](https://doc.rust-lang.org/std/ops/index.html#traits) and [`std::cmp`](https://doc.rust-lang.org/std/cmp/index.html#traits).
+
+For example, if you define a custom type representing a filesystem path[^1], you might implement path joining by means of the `/` operator by implementing [`std::ops::Div`](https://doc.rust-lang.org/std/ops/trait.Div.html).
```

Comment:
although the footnote doesn't work in GitHub's view, shall I assume that it will work on the Exercism website and approve anyway? https://github.com/exercism/v3/blob/a5308ec1ca019deb6a40aa35b33630fa769796d5/languages/rust/concepts/numbers/about.md

I see that this footnote syntax is in compliance with https://www.markdownguide.org/extended-syntax/, so I guess I'll hope that the website knows.

Reply:
Good point. I don't know whether the exercism markdown renderer supports footnote syntax. @iHiD do you know?

Reply:
We're using the [CommonMarker gem](https://www.gjtorikian.com/commonmarker/) to render markdown. As per its docs:

> Ruby wrapper for libcmark-gfm, GitHub's fork of the reference parser for CommonMark. It passes all of the C tests, and is therefore spec-complete. It also includes extensions to the CommonMark spec as documented in the GitHub Flavored Markdown spec, such as support for tables, strikethroughs, and autolinking.

The `:FOOTNOTES` option is currently not enabled as far as I could tell. @iHiD is that correct?

Reply:
I think we can turn it on. I'll try closer to the time. If it turns out that we can't for whatever reason, then we can fix this "manually" then? (ie, happy to merge)

Reply:
Great, thanks!